### PR TITLE
Introduced secrets plugin in the plugin section

### PR DESCRIPTION
**Pull request by maheshp**

### CODE REVIEW

File: data/plugins/secrets_plugins.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- name: GoCD file based Secrets Plugin
+  description: Bundled plugin which allows secrets to be stored/retrived from an external file.
+  readmore_url: https://github.com/gocd/gocd-file-based-secrets-plugin 
+  author_url: http://www.thoughtworks.com/products/
+  author_name: GoCD Contributors
+  releases_url: https://github.com/gocd-contrib/docker-elastic-agents/releases
```
Comment (kritika-singh3) [inline data/plugins/secrets_plugins.yml]: Update this?

File: source/plugins.html.erb [hunk 2]
```diff
@@ -51,6 +51,7 @@ meta_keywords: "GoCD, continuous delivery, software, continuous integration, ope
         {id: 'notification', kind: "Notification plugins", plugins: data.plugins.notification_plugins},
         {id: 'package-repo', kind: "Package repository plugins", version_info: ">= GoCD v13.3", plugins: data.plugins.repo_plugins},
         {id: 'scm', kind: "SCM plugins", plugins: data.plugins.scm_plugins},
+        {id: 'secrets', kind: "Secrets plugins", plugins: data.plugins.secrets_plugins},
```
Comment (kritika-singh3) [inline source/plugins.html.erb]: should we say `>= GoCD 19.6.0`??

File: data/plugins/secrets_plugins.yml (hunk 1 above)
Comment (maheshp) [reply data/plugins/secrets_plugins.yml]: Done, sorry missed during copy paste.

File: source/plugins.html.erb (hunk 2 above)
Comment (maheshp) [reply source/plugins.html.erb]: yes, done.

**Pull request merged**